patches/ios-chrome-app-BUILD.gn.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
+index 6eb60dddf6cd7fe5a87b2e463187db09a2b5d7cb..71bd4490f4eb31163557e2479a014047bd1e170c 100644
+--- a/ios/chrome/app/BUILD.gn
++++ b/ios/chrome/app/BUILD.gn
+@@ -166,7 +166,7 @@ source_set("credential_provider_migrator_app_agent") {
+     "//components/keyed_service/core",
+     "//ios/chrome/app/application_delegate:app_state_header",
+     "//ios/chrome/app/application_delegate:observing_app_state_agent",
+-    "//ios/chrome/browser/credential_provider",
```

Comment:
these patches are definitely not ok and I don't see a follow-up to fix them

Reply:
I see https://github.com/brave/brave-core/pull/9558/commits/fd9f14d2e902a0fa6a0ee13b5a7697c446d90699 but I don't see that these patches are removed

Reply:
Reverted this one and changed it to upstream patch from https://chromium.googlesource.com/chromium/src.git/+/ab3809244ea889dd3232708c700893e230a559ea, this patch will go away in cr95. Fixed in 1ce5ef0aa82cde44b1b0a941cdb7fecb9a4762ee